deps/config/manifests/store.yml.template

```diff
@@ -58,9 +58,9 @@
       address: http://hashicorp:8200
       token: '{VAULT_TOKEN}'
       namespace: ''
-- kind: Eth1Account
+- kind: EthAccount
```

Comment:
I would rather rename this to "Ethereum" to correlate with the rest of the changes

Reply:
applied